### PR TITLE
feature: Basic inputs

### PR DESCRIPTION
**Pull request by mhomolak**

### CODE REVIEW

File: scss/elements/inputs/_basic-input-mixins.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    @extend .abstract;
+    border: $textInput-basic-error-border;
+
+    @include pe-placeholder {
+      font-size : $textInput-placeholder-font-size;
+      color     : $textInput-placeholder-color;
+    }
+
+    &:focus {
+      outline: none;
```
Comment (StommePoes) [inline scss/elements/inputs/_basic-input-mixins.scss]: I wonder if we should keep the focus outline but make it 1px solid and match the box-shadow colour to be safe? There was an instance where a supported browser did not allow box shadow (though I think that was iOS with select boxes).

**Comment by StommePoes** [comment]
Other than my 1 comment, looks good.

File: scss/elements/inputs/_basic-input-mixins.scss (hunk 1 above)
Comment (mhomolak) [inline scss/elements/inputs/_basic-input-mixins.scss]: http://caniuse.com/#feat=css-boxshadow

It looks like it should be good to use now.

Comment (wyseguyonline) [inline scss/elements/inputs/_basic-input-mixins.scss]: If there are supported browsers that don't allow box shadow, IIRC there is a polyfill for that (worst case scenario of course).

**Pull request merged**